src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,42 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+  const stringArray = number.toString().split("").reverse();
+  const numberArray = stringArray.map(Number);
+  return numberArray;
+};
 
 module.exports = numberToReversedDigits;
+
+
+/*
```

Comment:
Its good you are taking notes as you go, so youre aware standard practice ios to not include notes like this unless they are for methods that need some detailed explanation that isnt available anywhere else, in this case these are standard javascript methods and as such there is no lack of docs for them. By all means take notes, but Id advise not using the code as the place to put them in cases like this. Just something to be aware of :)